nova/templates/etc/_api_audit_map.conf.tpl

```diff
@@ -0,0 +1,72 @@
+[DEFAULT]
+# default target endpoint type
+# should match the endpoint type defined in service catalog
+target_endpoint_type = None
```

Comment:
i assumed the files in the pycadf etc folder to be ready to go, but you might be right, that this needs adjustment - will check